options.js

```diff
@@ -4,11 +4,36 @@ let removeButton = document.getElementById("removeButton");
 const input = document.getElementById("colorInput");
 const confirmationMessage = document.getElementById("confirmationMessage");
 const maxMenuSize = 10;
+const skipIntroCheckBox = document.getElementById("skipIntroCheckbox");
+const spanIntroCheckBox = document.getElementById("spanIntroCheckBox");
+let extensionColor = null;
 
 getExtensionColor().then(color => updateExtensionColor(color));
 getColorMenu().then(colorOptions => buildButtons(colorOptions));
+getIntroFeatureState().then(state => skipIntroCheckBox.checked = state);
+
+function setCheckBoxColor() {
+  if(skipIntroCheckBox.checked) {
+    spanIntroCheckBox.style.backgroundColor = extensionColor;
+  } else {
+    spanIntroCheckBox.style.backgroundColor = chineseSilver;
+  }
+}
+
+skipIntroCheckBox.onclick = () => {
+  setCheckBoxColor();
+  setIntroFeatureState(skipIntroCheckBox.checked);
+}
+
+function setIntroFeatureState(state) {
+  chrome.storage.sync.set({ isIntroFeatureActive: state }, function () {
```

Comment:
What is the default value, when it's not set?

Reply:
The default value is true (in the beginning the checkbox is already marked). The value is set when the user clicks the checkbox for the first time.